node/src/BaseClient.ts

```diff
@@ -6241,8 +6254,14 @@ export class BaseClient {
      * console.log(result); // Output: 'acd'
      * ```
      */
-    public async lcs(key1: string, key2: string): Promise<string> {
-        return this.createWritePromise(createLCS(key1, key2));
+    public async lcs(
+        key1: GlideString,
+        key2: GlideString,
+        decoder?: Decoder,
```

Comment:
Please use decoder option
```suggestion
        options?: DecoderOption,
```
And update the doc accordingly